fastly/kinesis.go

```diff
@@ -26,7 +26,7 @@ type Kinesis struct {
 	DeletedAt         *time.Time `mapstructure:"deleted_at"`
 }
 
-// kinesesByName is a sortable list of Kineses.
+// kinesesByName is a sortable list of Kinesis.
```

Comment:
```suggestion
// kinesisByName is a sortable list of Kinesis.
```
And in the type below :)

Reply:
Nice spot!